schemas/components/stepProgress.yaml

```diff
@@ -0,0 +1,18 @@
+StepProgress:
```

Comment:
The boundary between `component` and `response` are kind blur. Maybe we can merge the two directories into one(since all the `yaml`s in these directories are for response). Also if one `component` is specific to another, like here `StepProgress` to `progressResult`, can we put them in one file(to reduce the number of files), just like we put `ClusterModelStatsMetaData` etc. in `ClusterModelStats.yaml`

Reply:
I'll put it all in `responses`, thanks!